CSC.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+title: "中国学生社区 (Chinese Student Community)"
+permalink: /CSC/
```

Comment:
Don't use capital char in permalink, keept them consistent with the existing undercase_snake_format.

Reply:
also, for CSC and HCC images directory, it's also highly recommended to use undercase_snake_format